@types/gql.d.ts

```diff
@@ -47,13 +47,15 @@ type UpdateProjectArgs = {
 type ProjectQuery = {
```

Comment:
At some point, we should move `Interfaces.tsx` to the `@types` directory and rename the file to use the `.d.ts` extension. This file also defines a `ProjectQuery` type. I'm not sure on the differences in use or if two types are even needed. If we don't address this now, please create an issue for this item.

Reply:
Opened #60